styles.css

```diff
@@ -0,0 +1,1011 @@
+  justify-content: center;
+  align-items: baseline;
+  margin-left: -35px;
+  margin-top: -25px;
+  margin-bottom: -30px;
+}
+
+#sqr {
+  width: 18px;
+  height: 18px;
+  margin-right: 20px;
+  transform: rotate(-45deg);
+}
+
+#circle {
+  width: 18px;
+  height: 18px;
+  margin-right: 20px;
+}
+
+#skill {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: baseline;
+  margin-left: 55px;
+}
+
+.contact {
+  background: #fff;
+  width: 327px;
+  height: 720px;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact p {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  padding: 51px 24px;
+  width: 327px;
+  height: 216px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: -30px;
+}
+
+#full_name {
+  text-align: center;
+  height: 18px;
+  width: 360px;
+}
+
+#form {
+  width: 327px;
+  height: 272px;
+}
+
+#message {
+  background-color: #fbf8f7;
+  height: 114px;
+  width: 310px;
+  padding: 10px 24px;
+  font-family: 'Inter', sans-serif;
+  margin-left: 10px;
+}
+
+#divider1 {
+  border: 1px solid #979493;
+  width: 320px;
+  margin: 20px 20px;
+}
+
+::placeholder {
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+}
+
+#getintouch_btn {
+  margin-left: 117px;
+  margin-top: 22px;
+  padding-left: 10px;
+  font-family: 'Inter', sans-serif;
+}
+
+#getintouch_btn:hover {
+  background-color: #cc7333;
+}
+
+.social {
+  width: 375px;
+  display: flex;
+}
+
+#end {
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  width: 320px;
+}
+
+#divider2 {
+  border: 1px solid #979493;
+  width: 320px;
+  margin: 18px 20px;
+}
+
+.desktop_headicons {
+  display: none;
+}
+
+.desktop_projects {
+  display: grid;
+  align-items: center;
+  width: 375px;
+  margin: 0.1px 20px;
+}
+
+.project_descp1 {
+  width: 295px;
+  height: 81px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  position: relative;
+  top: 56px;
+  left: 16px;
+}
+
+.project_detail {
+  width: 295px;
+  height: 87px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  position: relative;
+  top: 38px;
+  left: 16px;
+}
+
+.project {
+  background-image: url(/attributes/work-image-1.png);
+  background-repeat: round;
+  width: 327px;
+  height: 386px;
+  position: relative;
+}
+
+@media only screen and (min-width: 769px) {
+  body {
+    background-color: #e5e5e5;
+    margin: 0%;
+  }
+
+  .container-header {
+    width: 100%;
+    height: 72px;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .desktop_headicons {
+    display: flex;
+    margin-left: auto;
+    margin-right: 150px;
+    gap: 25px;
+    align-items: center;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
```

Comment:
[OPTIONAL] As you are trying to build a responsive website, you should ensure your website looks as like the figma design at all screen width and media break points as much as possible. You can consider adding more media breakpoints if it is difficult to achieve this with only one. :pray: 
Certain parts of your website does not look responsive for screen widths between 768px to 1150px as shown in the images below. Kindly check and rectify accordingly to help make your website more responsive. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 07-57-44](https://user-images.githubusercontent.com/65233860/199926674-a39c3a60-ef85-4154-bd55-cd22af521492.png)  |  ![Screenshot from 2022-11-04 08-16-28](https://user-images.githubusercontent.com/65233860/199926737-ecf324c3-6b68-4513-b7e8-673ed48e823d.png)

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 07-58-02](https://user-images.githubusercontent.com/65233860/199926954-2ac3cb4d-0c6a-4685-8eea-1741220228be.png)  |  ![Screenshot from 2022-11-04 08-32-02](https://user-images.githubusercontent.com/65233860/199928085-4a5380ec-c556-4e03-81e9-38688dec49a1.png)


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-04 08-29-44](https://user-images.githubusercontent.com/65233860/199927564-a9b364c3-a2f8-4ab3-875a-6f7d16d13e96.png)  |  ![Screenshot from 2022-11-04 08-16-51](https://user-images.githubusercontent.com/65233860/199927605-c60c58e7-917c-4a57-a814-28e055e20d9e.png)